## MBZIRC detector configuration of ROS parameters

subscribers:

    input_camera:
        topic: /mux_cam/image_raw


publishers:
    
    # Note: found_object not implemented yet
    found_object:
        enable: false
        topic: /darknet_ros/found_object
        queue_size: 1
        latch: false

    bounding_boxes:
        enable: true
        topic: /darknet_ros/bounding_boxes
        queue_size: 1
        latch: true

    detection_image:
        enable: true
        topic: /darknet_ros/detection_image
        queue_size: 1
        latch: true


cameras:

    long_dist:
        name: Mobius_ActionCam
        topic: /mobius_cam/image_raw
        stereo: false

    short_dist:
        name: Ausdom_Webcam
        topic: /ausdom_cam/image_raw
        stereo: false

## Use these when stereocamera support is available
#       name: Zed_StereoCam
#       topic: /zed_cam/image_raw
#       stereo: true

